akka-remote/src/main/scala/akka/remote/serialization/MessageContainerSerializer.scala

```diff
@@ -31,6 +26,8 @@ class MessageContainerSerializer(val system: ExtendedActorSystem) extends BaseSe
   def includeManifest: Boolean = false
```

Comment:
Doesn't this have to includeManifest now?

I would also prefer to use `SerializerWithStringManifest` because then we don't need to send the full class name in each message. Perhaps you have seen how I have used short manifest names such as "A", "B" in other new serializers.

We could probably get this `MessageContainerSerializer` working as a SerializerWithStringManifest in a wire compatible way, but it's risky business and not worth the effort in my opinion. I think we should create a completely new serializer and leave `MessageContainerSerializer` alone. We know we will need more of those so perhaps we should name the new serializer in a way that it can handle various messages. E.g. `MiscMessageSerializer`.